charts/kyverno-policies/templates/baseline/disallow-capabilities.yaml

```diff
@@ -43,9 +43,26 @@ spec:
       exclude:
         {{- toYaml . | nindent 8 }}
       {{- end }}
-      {{- with index .Values "policyPreconditions" $name }}
+      {{- $preconditions := index .Values "policyPreconditions" $name }}
```

Comment:
Can't understand that line. Why we need it ?

Reply:
The value for the precondition is checked in several places so needs to be assigned to a variable to reduce duplication.  This same pattern is already used in policies like here: https://github.com/kyverno/kyverno/blob/295e98aebe2abe810e9cd741d218c6e4da6564c7/charts/kyverno-policies/templates/restricted/disallow-capabilities-strict.yaml#L47.

The variable name has a number if the above code because that template has multiple rules so multiple preconditions.  The changes in this PR are policies with only 1 rule so the number in variable isn't needed.

Reply:
Got it. Thank you!